src/promptflow/promptflow/_telemetry/activity.py

```diff
@@ -6,6 +6,7 @@
 import uuid
 from datetime import datetime
 
+from promptflow._core.operation_context import OperationContext
```

Comment:
we might need to move the _telemetry folder to  _sdk @D-W-

Reply:
This PR will not be used as a mobile folder temporarily.